src/pegin/components/create/ConfirmTx.vue

```diff
@@ -0,0 +1,325 @@
+<template>
+ <div class="transactions">
+    <v-row class="mx-0 d-flex justify-center">
+      <v-col cols="10" lg="8" xl="6" class="d-flex justify-center">
+        <h1 class="text-center">Confirm transaction on your wallet</h1>
+      </v-col>
+    </v-row>
+    <v-row class="mx-0 my-8 d-flex justify-center">
+      <p class="text-center">
+        Make sure the amount, address and transaction fee displayed
+        on the {{ walletService.name() }} wallet are correct.
+        <br>
+        To prevent malware attacks, double-check the address with the recipient.
+        <br>
+        Press <strong>Send</strong> when you finish.
+      </p>
+    </v-row>
+    <v-row justify="center" class="mb-5">
+      <v-col v-for="step, idx in walletService.confirmationSteps()" :key="`step-${idx}`"
+          class="step" >
+        <v-row class="step-title">
+          <v-col>
+            <v-row class="mx-0 d-flex justify-center">
+              <v-img :src="require('@/assets/exchange/rootstock_black.png')" height="40" contain/>
+            </v-row>
+            <v-row class="mx-0 my-3 d-flex justify-center">
+              <h4 class="text-center">
+                <div v-if="walletService.confirmationSteps().length > 1"
+                  class="number">{{ idx + 1 }}</div>
+                {{ step.title }}
+              </h4>
+            </v-row>
+          </v-col>
+        </v-row>
+        <fieldset class="confirmation-box">
+          <legend lass="px-4">See on {{ walletService.name() }}</legend>
+          <v-row class="ma-0 text-center">
+            <v-col>
+              <v-row justify="center" class="mt-3 mb-2">
+                <span>
+                  {{ step.subtitle }}
+                </span>
+              </v-row>
+              <v-row v-if="step.outputsToshow.opReturn.amount || step.outputsToshow.opReturn.value"
+                 class="mt-2 d-flex justify-center">
+                <span>
+                  OP_RETURN
+                </span>
+                <v-tooltip right>
+                  <template v-slot:activator="{props}">
+                      <v-icon v-bind="props" v-on="props.on" color="grey-lighten-1">
```

Comment:
The icon needs to be imported from the `@mdi/js` package to be properly displayed.